frontend/app/src/theme/community/windoge98.ts

```diff
@@ -0,0 +1,107 @@
+import type { Theme } from "../types";
+import { hexPercent } from "../utils";
+
+const textBox = "rgba(226,226,226,0.4)";
+const txt = "#000000";
+const txt40 = hexPercent(txt, 40);
+const txt60 = hexPercent(txt, 60);
+const txt80 = hexPercent(txt, 80);
+const disabledTxt = txt40;
+const white = "#FDFFFF";
+const black = "#000000";
+const silver = "#C0C0C0";
+const trolleyGray = "#818181";
+const msBlue = "#010081";
+const darkGray = "#696969";
+const magenta = "#ff0081";
+const teal = "#008080";
+const successGreen = "#28a745";
+const errorRed = "#dc3545";
+
+export function getTheme(base: Theme): Theme {
+    base.author = "rh2pm-ryaaa-aaaan-qeniq-cai";
+    base.name = "windoge98";
+    base.label = "Windoge98";
+    base.bg = "#008080 url('/assets/windoge98logo.png') bottom 60px right 10px no-repeat";
+    base.burst = false;
+    base.logo = false;
+    base.txt = txt80;
+    base["txt-light"] = txt60;
+    base.bd = silver;

```

Comment:
In some cases the borders cannot be seen using this colour. I would suggest something a whisker darker like #b0b0b0 maybe.